src/lightning_app/components/multi_node/executors.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from lightning_app.utilities.proxies import WorkRunExecutor
+
+
+class LiteRunExecutor(WorkRunExecutor):
+    def __call__(
+        self,
+        main_address: str,
+        main_port: int,
+        num_nodes: int,
+        node_rank: int,
+    ):
+        os.environ["MASTER_ADDR"] = main_address
+        os.environ["MASTER_PORT"] = str(main_port)
+        os.environ["NODE_RANK"] = str(node_rank)
+
+        lite = self.configure_lite(num_nodes)
+        lite.launch(function=self.work_run)
+
+    def configure_lite(self, num_nodes: int):
+        from lightning.lite import LightningLite
+
+        return LightningLite(accelerator="auto", devices="auto", strategy="ddp_spawn", num_nodes=num_nodes)
```

Comment:
This is good for now, I'm wondering if there's a point in exposing `accelerator` and `devices` (to `MultiNode` or to the executor class) so that a user can customize them.

Reply:
I decided to expose Lite entirely.